### PR TITLE
Feature/adding simple linker

### PR DESCRIPTION
**Pull request by tomas-knap**
Simple Silk linker (with Silk called as ext process)

Needed by @jindrichmynarz to finish certain pipelines.

### CODE REVIEW

**Comment by tomas-knap** [comment]
Please add SK About and SK localization based on EN About/localization

**Comment by tomas-knap** [comment]
@skrchnavy  Please assign somebody who will finish SK about/loc

File: e-silkLinker/pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    <artifactId>uv-e-silkLinker</artifactId>
+    <name>E-SilkLinker</name>
+    <description>Creates links between RDF resources based on the Silk Link Specification Language (LSL),
+        https://www.assembla.com/spaces/silk/wiki/Link_Specification_Language.
+        The script may be uploaded/adjusted in the DPU configuration. Output section of such script is always ignored,
+        output is written to two output data units of the DPU - "links_confirmed", "links_to_be_verified".
+        DPU configuration may also specify thresholds for the two outputs created. Uses 2.5.3 version of Silk. Not
+        supporting cancelation of the DPU.
+    </description>
+    <version>2.0.0-SNAPSHOT</version>
```
Comment (skrchnavy) [inline e-silkLinker/pom.xml]: is version OK (to start with 2.0.0?)

File: e-silkLinker/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>eu.unifiedviews</groupId>
+            <artifactId>uv-dpu-helpers</artifactId>
+            <scope>compile</scope>
+        </dependency>
+
+
+        <!--
```
Comment (skrchnavy) [inline e-silkLinker/pom.xml]: unused part shall be removed

File: e-silkLinker/pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <packaging>bundle</packaging>
+
+    <properties>
+        <dpu.package>eu.unifiedviews.plugins.extractor.silklinker</dpu.package>
+        <dpu.mainClass>SilkLinker</dpu.mainClass>
+    </properties>
+
+    <dependencies>
+                <!-- UnifiedViews helpers. -->
+        <dependency>
```
Comment (skrchnavy) [inline e-silkLinker/pom.xml]: this shall be used from parent pom (uv-pom-dpu) so shall be removed.

File: e-silkLinker/pom.xml (hunk 1 above)
Comment (tomas-knap) [inline e-silkLinker/pom.xml]: Originally we had Silk linker 1.X for UV 1.X, so lets go for 2.0.0

File: e-silkLinker/pom.xml (hunk 3 above)
Comment (tomas-knap) [inline e-silkLinker/pom.xml]: The particular version of helpers is provided by parent, but the dependency has to be there in order to compile the project

Comment (skrchnavy) [inline e-silkLinker/pom.xml]: OK, I see. Candidate for later improvement....

**Comment by tomas-knap** [comment]
resources.properties SK is missing

**Comment by skrchnavy** [comment]
@mkl-eea-sk  to finalize slovak version

**Comment by mkl-eea-sk** [comment]
I have added margin to Vaadin dialog, so it does not look like this:
silk_margin_false
but to look like this:
silk_margin_true

the slovak version was added already by @mbl-eea-sk in https://github.com/UnifiedViews/Plugins/commit/9ba33e0d338967cd6a3b56cfadd09d2e78cd4a62

**Pull request merged**